pageserver/src/layered_repository/delta_layer.rs

```diff
@@ -38,23 +38,25 @@
 //! parts: the page versions and the relation sizes. They are stored as separate chapters.
 //!
 use crate::layered_repository::blob::BlobWriter;
-use crate::layered_repository::filename::DeltaFileName;
+use crate::layered_repository::filename::{DeltaFileName, PathOrConf};
 use crate::layered_repository::storage_layer::{
     Layer, PageReconstructData, PageVersion, SegmentTag,
 };
 use crate::repository::WALRecord;
+use crate::waldecoder;
 use crate::PageServerConf;
 use crate::{ZTenantId, ZTimelineId};
 use anyhow::{bail, Result};
 use bytes::Bytes;
 use log::*;
 use serde::{Deserialize, Serialize};
 use std::collections::BTreeMap;
+use std::fmt::Write as FmtWrite;
```

Comment:
Note that this trait is not used by name, but is needed for `write!`. We rename it so it doesn't conflict with the `std::io::Write` import. Do you know of a better practice for this @sharnoff ?

Reply:
@patins Yup! For things like this, you can write `use std::fmt::Write as _;` - it does the thing you expect. (Although it's possibly worth a small comment along the lines of: "avoid binding to `Write` again, just importing to get its methods"

There's also a section in the reference about importing with underscore with some neat stuff in there: https://doc.rust-lang.org/reference/items/use-declarations.html#underscore-imports